api/swagger.yaml

```diff
@@ -7202,6 +7202,11 @@ paths:
             - `reference`=(`<image-name>[:<tag>]`)
             - `since`=(`<image-name>[:<tag>]`,  `<image id>` or `<image@digest>`)
           type: "string"
+        - name: "shared-size"
+          in: "query"
+          description: "Show shared size as a `SharedSize` field on each image."
```

Comment:
(thinking out loud) wondering if we should mention `compute` here and/or some mention that setting this option will introduce additional overhead. I see we're not doing this (currently) for some other `size` parameter, but maybe we should.

Reply:
I also think `compute` is a better term here, but I went with "show" for consistency.
Perhaps we should revise all of these in a separate PR?